pkg/controllers/resources/nodes/syncer.go

```diff
@@ -35,15 +37,27 @@ func NewSyncer(ctx *synccontext.RegisterContext, nodeService nodeservice.NodeSer
 		}
 	}
 
+	// parse tolerations
+	var tolerations []*corev1.Toleration
+	if len(ctx.Options.Tolerations) > 0 {
+		for _, t := range ctx.Options.Tolerations {
+			tol, err := toleration.ParseToleration(t)
+			if err == nil {
```

Comment:
What happens when the error is not nil? How are we handling any error that occurs when toleration.ParseToleration() function is called?